src/remote_adapter/remote_adapter_client/CMakeLists.txt

```diff
@@ -0,0 +1,108 @@
+project(remote)
+cmake_minimum_required(VERSION 3.11)
+include(FetchContent)
+
+#
+# Options
+#
+option(BUILD_WITH_CLIENT_LOGGING
+  "ALlow logging in the library for debug purposes."
+  OFF)
+
+# Enable PIC as static libraries may be linked to shared objects
+set (CMAKE_POSITION_INDEPENDENT_CODE TRUE)
+set (CMAKE_CXX_STANDARD 11)
+set (CMAKE_INCLUDE_CURRENT_DIR ON)
+set (CMAKE_AUTOMOC ON)
+
+FetchContent_Declare(
+  rpclib
+  GIT_REPOSITORY https://github.com/dboltovskyi/rpclib.git
+  GIT_TAG        v2.2.2
+)
+
+FetchContent_GetProperties(rpclib)
+if(NOT rpclib_POPULATED)
+  FetchContent_Populate(rpclib)
+  add_subdirectory("${rpclib_SOURCE_DIR}" "${rpclib_BINARY_DIR}")
+  find_package( Threads REQUIRED )
+  add_library(rpc::rpclib ALIAS rpc)
+endif()
+
+find_package(Qt5 COMPONENTS Core REQUIRED)
+find_package( Threads REQUIRED )
+find_package(Lua 5.2 EXACT REQUIRED)
+
+list(GET LUA_LIBRARIES 0 LUA_LIB)
+list(GET LUA_LIBRARIES 1 LIBM_LIB)
+message(STATUS "lualib : ${LUA_LIB}")
+message(STATUS "libmlib : ${LIBM_LIB}")
+
+if (NOT TARGET lua::lua)
+  add_library(lua::lua UNKNOWN IMPORTED GLOBAL)
+
+  set_target_properties(lua::lua
+    PROPERTIES
+      INTERFACE_INCLUDE_DIRECTORIES "${LUA_INCLUDE_DIR}"
+      IMPORTED_LOCATION ${LUA_LIB}
+      IMPORTED_LINK_DEPENDENT_LIBRARIES ${LIBM_LIB}
+      )
+endif()
+
+set(RPCLIB_DEPENDENCIES "${rpclib_SOURCE_DIR}/dependencies")
+file(GLOB_RECURSE DEP_HEADERS
+  ${RPCLIB_DEPENDENCIES}/include/*.h
+  ${RPCLIB_DEP_LIBRARIDEPENDENCIES}/include/*.hpp)
```

Comment:
@jacobkeeler This variable is variable of third party library `rpclib` which is used in current implementation of remote ATF. So unfortunately  we can not rename it to fix the typo.

Reply:
Sorry for the delayed response, but I wasn't able to test until now. This variable is empty on my machine, so I think it can be safely removed from this list.

Reply:
@jacobkeeler 
Fixed in: https://github.com/smartdevicelink/sdl_atf/pull/180/commits/39d4cd0ef42cb5ec97f43301499c8201f95a09f7